packages/cubejs-client-core/src/ResultSet.js

```diff
@@ -537,15 +537,15 @@ class ResultSet {
         let currentItem = schema;
 
         yValues.forEach((value, index) => {
-          currentItem[value] = {
+          currentItem['_' + value] = {
```

Comment:
could you also fix the `eslint` issues?

Reply:
Done, no idea why my editor didnt show it before. just started it to fix and it showed the error :D